lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ContinuousRankedProbabilityScoreMinimisers.py

```diff
@@ -419,8 +418,7 @@ def setUp(self):
         self.truth_data = self.truth.data.flatten().astype(np.float64)
 
 
-class Test_truncated_normal_crps_minimiser(
-        SetupTruncatedGaussianInputs, EnsembleCalibrationAssertions):
+class Test_truncated_normal_crps_minimiser(SetupTruncatedGaussianInputs):
```

Comment:
Again should the name be changed because it is no longer minimising the CRPS?

Reply:
I've renamed the function and associated unit tests to cover this.

Reply:
Great, thanks @gavinevans